frontend/packages/console-shared/src/selectors/namespace.ts

```diff
@@ -0,0 +1,5 @@
+import * as _ from 'lodash';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+export const getRequester = (obj: K8sResourceKind) =>
+  _.get(obj, ['metadata', 'annotations', 'openshift.io/requester']);
```

Comment:
We already have this helper

https://github.com/spadgett/console/blob/a1b91bf1c0f88a79f1fc715d1ba91cea4c11a20d/frontend/public/components/namespace.jsx#L65

If you're going to move it to selectors, you should remove the old one.